# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/xterm-ls.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

A new xTerm should (= must!) have a login shell.

  - Clifford, June 2000

--- ./programs/xterm/XTerm.ad~	2003-11-02 05:46:48.000000000 +0100
+++ ./programs/xterm/XTerm.ad	2003-11-19 21:49:00.000000000 +0100
@@ -110,3 +110,7 @@
 *tek4014*font2: 8x13
 *tek4014*font3: 6x13
 *tek4014*fontSmall: 6x10
+
+! A new xTerm should (= must!) have a login shell.  (Clifford)
+*loginShell: on
+
